src/routes/api-documentation.ts

```diff
@@ -1,106 +1,107 @@
-              [hydra.title]: { '@value': 'What the collection manages', '@language': 'en' },
-              [hydra.property]: {
-                '@id': 'http://www.w3.org/ns/hydra/core#manages',
-                '@type': rdf.Property,
-              },
-              [hydra.readable]: true,
-              [hydra.required]: true,
-              [hydra.writeable]: false,
-            },
-            {
-              '@type': hydra.SupportedProperty,
-              [hydra.title]: { '@value': 'Total items', '@language': 'en' },
-              [hydra.property]: {
-                '@id': hydra.totalItems,
-                '@type': rdf.Property,
-              },
-              [hydra.readable]: true,
-              [hydra.required]: true,
-              [hydra.writeable]: false,
-            },
-            {
-              '@type': hydra.SupportedProperty,
-              [hydra.title]: { '@value': 'Members of this collection', '@language': 'en' },
-              [hydra.property]: {
-                '@id': hydra.member,
-                '@type': [rdf.Property, hydra.Link],
-              },
-              [hydra.readable]: true,
-              [hydra.required]: false,
-              [hydra.writeable]: false,
-            },
-          ],
-        },
-      ],
-    };
-    response.type = 'jsonld';
+    const apiDocumentation = namedNode(
+      url.resolve(request.origin, router.url(Routes.ApiDocumentation)),
+    );
+    const entryPoint = namedNode(url.resolve(request.origin, router.url(Routes.EntryPoint)));
+
+    const quads = [
+      quad(apiDocumentation, rdf('type'), hydra('ApiDocumentation')),
+      quad(apiDocumentation, hydra('entrypoint'), entryPoint),
+    ];
+
+    // schema:EntryPoint
+    const entryPointGet = blankNode();
+    const entryPointName = blankNode();
+    quads.push(
+      quad(apiDocumentation, hydra('supportedClass'), schema('EntryPoint')),
+      quad(schema('EntryPoint'), rdf('type'), hydra('Class')),
+      quad(schema('EntryPoint'), hydra('title'), literal('API entry point', 'en')),
+      quad(schema('EntryPoint'), hydra('supportedOperation'), entryPointGet),
+      quad(entryPointGet, rdf('type'), hydra('Operation')),
+      quad(entryPointGet, hydra('title'), literal('Get the entry point', 'en')),
+      quad(entryPointGet, hydra('method'), literal('GET')),
+      quad(entryPointGet, hydra('expects'), owl('Nothing')),
+      quad(entryPointGet, hydra('returns'), schema('EntryPoint')),
+      quad(schema('EntryPoint'), hydra('supportedProperty'), entryPointName),
+      quad(entryPointName, rdf('type'), hydra('SupportedProperty')),
+      quad(entryPointName, hydra('title'), literal('Name', 'en')),
+      quad(entryPointName, hydra('property'), schema('name')),
+      quad(schema('name'), rdf('type'), rdf('Property')),
+      quad(entryPointName, hydra('required'), toRdf(true)),
+      quad(entryPointName, hydra('readable'), toRdf(true)),
+      quad(entryPointName, hydra('writeable'), toRdf(false)),
+    );
```

Comment:
Clearly this isn't that easy to read.

Wondering if it could be a Turtle (or similar) string that gets parsed. Will leave for now though.